docs/chain-details/module_distribution.md

```diff
@@ -0,0 +1,119 @@
+### `distribution` module
+
+### Introduction
+
+The `distribution` module is responsible for the distribution of rewards to the validators and delegators.
+
+### Overview
+
+#### Network Parameters
+
+Below are all the network parameters for the `distribution` module:
+
+- `community_tax` - The rate of community tax;
```

Comment:
I guess in this module description, I guess it'll be good to explain/note why community pool may be non-zero even if the tax rate is zero.

Reply:
Added further details on [Community tax](https://github.com/crypto-com/chain-docs/pull/38/commits/89c4e43c18b989be0b84687afa780187dfe932ee#diff-908b614c2992c840fd286f4c491cc499dd1cbf5b52bc3a868d1adf6b3ffdbf8fR46).